### PR TITLE
fix: fix release workflow

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Fix release process, after moving dockerfiles inside the tests/e2e sources directory
Add npm publish to release builds

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: make-release.sh [hunk 1]
```diff
@@ -156,12 +156,10 @@ echo "${VERSION}" > VERSION
...
 sed_in_place -r -e "/@eclipse-che\/api|@eclipse-che\/workspace-client|@eclipse-che\/workspace-telemetry-client/!s/(\"@eclipse-che\/..*\": )(\".*\")/\1\"$VERSION\"/" package.json
 npm --no-git-tag-version version --allow-same-version "${VERSION}"
+npm publish --tag "latest"
 popd >/dev/null || exit
 
-echo "Copying source code to dockerfile directory"
-cp -r "tests/e2e" "dockerfiles/e2e/e2e"
-
-docker build -t quay.io/eclipse/che-e2e:${VERSION} dockerfiles/e2e/
+docker build -t quay.io/eclipse/che-e2e:${VERSION} .
```
Comment (nickboldt) [inline make-release.sh]: there is no Dockerfile in the root. Do you mean this file?
https://github.com/eclipse/che/blob/main/tests/e2e/build/dockerfiles/Dockerfile

Comment (mkuznyetsov) [reply make-release.sh]: thx, fixed

File: make-release.sh [hunk 2]
```diff
@@ -158,10 +158,7 @@ sed_in_place -r -e "/@eclipse-che\/api|@eclipse-che\/workspace-client|@eclipse-c
 npm --no-git-tag-version version --allow-same-version "${VERSION}"
 popd >/dev/null || exit
 
-echo "Copying source code to dockerfile directory"
-cp -r "tests/e2e" "dockerfiles/e2e/e2e"
-
-docker build -t quay.io/eclipse/che-e2e:${VERSION} dockerfiles/e2e/
+docker build -t quay.io/eclipse/che-e2e:${VERSION} -f /build/dockerfiles/Dockerfile .
```
Comment (nickboldt) [inline make-release.sh]: `/build`? is this running inside a Dockerfile context?

File: make-release.sh (hunk 1 above)
Comment (nickboldt) [reply make-release.sh]: why not `tests/e2e/build/dockerfiles/Dockerfile` ?

File: make-release.sh (hunk 2 above)
Comment (nickboldt) [reply make-release.sh]: should this be `tests/e2e/build/dockerfiles/Dockerfile` ?

Comment (nickboldt) [reply make-release.sh]: and also all the files you want to compile are not in `.` but in `./tests/e2e/`

Comment (mkuznyetsov) [reply make-release.sh]: we did a pushd to tests/e2e eralier, so the path should be good, without the first slash?)

File: make-release.sh [hunk 3]
```diff
@@ -176,6 +173,8 @@ git commit -asm "${COMMIT_MSG}"
 git tag "${VERSION}"
 git push origin "${VERSION}"
 
+
```
Comment (nickboldt) [inline make-release.sh:176]: bonus empty lines?

File: make-release.sh [hunk 4]
```diff
@@ -158,10 +158,7 @@ sed_in_place -r -e "/@eclipse-che\/api|@eclipse-che\/workspace-client|@eclipse-c
 npm --no-git-tag-version version --allow-same-version "${VERSION}"
 popd >/dev/null || exit
 
-echo "Copying source code to dockerfile directory"
-cp -r "tests/e2e" "dockerfiles/e2e/e2e"
-
-docker build -t quay.io/eclipse/che-e2e:${VERSION} dockerfiles/e2e/
+docker build -t quay.io/eclipse/che-e2e:${VERSION} -f tests/e2e/build/dockerfiles/Dockerfile tests/e2e
```
Comment (nickboldt) [inline make-release.sh:161]: checking locally... works!

**Review by nickboldt — APPROVED** [review]

**Pull request merged**